app/reflexes/admins_table_reflex.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AdminsTableReflex < ApplicationReflex
+  include SortHelper
+
+  def sort
+    sort_records(records: User.where(role: :admin),
+      partial: "admins_table",
+      sort_columns: ["first_name", "last_name", "status"])
```

Comment:
Nice work with this refactoring! This looks like it will be an easy pattern to use on future pages (...students & tickets)